src/db/migrations/20180326231610_create-user_rankings-table.js

```diff
@@ -0,0 +1,25 @@
+const tableName = 'user_rankings'
+
+exports.up = knex => 
+  knex.schema.createTable(tableName, table => {
+    table.increments('id')
+    table.integer('item_id')
+      .notNullable()
+      .references('id')
+      .inTable('items')
+    table.integer('user_id')
+      .notNullable()
+      .references('id')
+      .inTable('users')
+    table.integer('prev_ranking_id')
+      .unique(),
+    table.timestamps(true, true)
+    table.timestamp('deleted_at')
```

Comment:
If we're allowing soft delete, we'll have to expand the unique constraint on prev_ranking_id since soft-deleted record can have the same prev_ranking_id as existing records. I'm not sure we gain much from allowing soft-delete anyway so maybe we can remove it?

Reply:
Do you think we should ditch it for everything, or just this table? I kinda agree that it doesn't do much for us here (haven't thought much about whether it's worthwhile elsewhere).

Reply:
Ah it looks like the other tables have the same issue with unique constraint + deleted_at. I thinks it's easiest to just drop deleted_at on all tables for now. We can revisit it later when we have a better idea of the data.